src/js/components/Anchor/Anchor.js

```diff
@@ -79,6 +79,8 @@ const Anchor = forwardRef(
       });
     }
 
+    coloredIcon = useSizedIcon(coloredIcon, sizeProp || size, theme);
```

Comment:
very minor, but "colored" doesn't seem like necessarily the right prefix given what we're doing now.

Reply:
maybe "anchorIcon", which would align with Button's "buttonIcon"?

Reply:
I'll go with `anchorIcon`